components/mpas-ocean/src/shared/mpas_ocn_tendency.F

```diff
@@ -424,7 +424,7 @@ subroutine ocn_tend_vel(tendPool, statePool, forcingPool, &
       ! Add forcing and bottom drag
       call ocn_vel_forcing_tend(normalVelocity, sfcFlxAttCoeff, &
                                 sfcStress, kineticEnergyCell, &
-                                layerThickEdge, tendVel, err)
+                                layerThickEdgeCenter, tendVel, err)
```

Comment:
My recollection from our conversation yesterday or last week was that we would likely want the upwind version in bottom drag because we would want bottom drag to be one of the terms that remains non-zero as we approach the thin film.  But I will defer to the folks you actually asked ;-)